### PR TITLE
WRAPR-60: send breaking from post

### PR DESCRIPTION
**Pull request by mogmarsh**
Allows you to create a newsletter from a regular post and send that newsletter when the post is published.

### CODE REVIEW

File: plugins/newsletter-from-post/email-settings.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+      return;
+    }
+    apiFetch({ path: '/newsletter-builder/v1/lists' }).then((response) => {
+      setLists(response as any as ListResult[]);
+    });
+  }, [lists]);
+
+  const updateNewsletter = () => {
+    if (posting) {
+      // setRetry(true);
```
Comment (kaitlinbolling) [inline plugins/newsletter-from-post/email-settings.tsx:135]: Needed?
```suggestion
```

File: plugins/newsletter-from-post/email-settings.tsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      return;
+    }
+    setPosting(true);
+    if (!postId) {
+      return;
+    }
+    if (!subject && !list && !content) {
+      return;
+    }
```
Comment (kaitlinbolling) [inline plugins/newsletter-from-post/email-settings.tsx:144]: Do you need to set posting to false in these cases?
```suggestion
    if (!postId) {
      setPosting(false);
      return;
    }
    if (!subject && !list && !content) {
      setPosting(false);
      return;
    }
```

**Review by kaitlinbolling — APPROVED** [review]
👍 looks good!

File: plugins/newsletter-from-post/email-settings.tsx (hunk 2 above)
Comment (mogmarsh) [reply plugins/newsletter-from-post/email-settings.tsx:147]: I just moved the `setPosting(true);` to after the returns.

**Pull request merged**